Goal: Add a consang.1 man page

Status wrt upstream: Patch proposed. Will be in 4.11?

Index: geneweb-4.10/man/consang.1
===================================================================
--- /dev/null	1970-01-01 00:00:00.000000000 +0000
+++ geneweb-4.10/man/consang.1	2005-08-30 08:20:16.000000000 +0200
@@ -0,0 +1,31 @@
+.TH GENEWEB 1 "2000 July 22"
+.UC 4
+.SH NAME
+consang \- Computer Genealogical Consanguinities
+.SH SYNOPSIS
+.B consang
+[
+.I file
+]
+.br
+.SH DESCRIPTION
+.I consant
+calculates the consanguinities in the
+.IR E Geneweb
+database passed at the command line.
+.br
+The displaying of consanguinities is done when consulting the
+database, in the personal records, and while computing
+relationships.
+.PP
+The primary documentation of
+.I consang
+ is in the
+.I HTML
+.IR  ,
+documentation, which you can find in the documentation directory (/usr/share/doc/geneweb on
+Debian systems).
+.br
+Please look there for complete and up-to-date documentation.
+.PP
+
